example/example.py

```diff
@@ -79,11 +80,56 @@ def webhook():
             response, error = bot.send_message(msg2, recipient)
 
         
+        elif m.text == 'receipt':
+            element = ReceiptElement(
+                title='My Title',
+                subtitle='A very good subtitle',
+                quantity='15',
+                price=1999,
+                currency='CAD',
+                image_url='http://www.newton.ac.uk/files/covers/968361.jpg'
+            )
+
+            receipt = Template(Template.receipt_type,
+                recipient_name= 'A name',
+                order_number='12345678902',
+                currency='CAD',
+                payment_method='Visa',
+                order_url='http://petersapparel.com/order?order_id=123456',
+                elements=[element],
+                address={
+                    'street_1': '1 Hacker Way',
+                    'city': 'Vancouver',
+                    'state': 'BC',
+                    'country': 'CA',
+                    'postal_code': '1A1A1A'
+
+                },
+                summary={
+                    'subtotal': 75.00,
+                    'shipping_cost': 4.95,
+                    'total_tax': 6.19,
+                    'total_cost': 56.14
+                }
+            )
+
+            msg = Message('template', receipt)
+            response, error = bot.send_message(msg, recipient)
+
         else: 
-            # Echo back to user
-            # Send regular 'text' message
             payload = text
+            # Create image message
+            msg = Message('image', 'http://www.buffettworld.com/images/news_trump.jpg')
+            # Create audio message
+            msg = Message('audio', 'http://www.stephaniequinn.com/Music/Canon.mp3')
+            # Create video message
+            msg = Message('video', 'http://clips.vorwaerts-gmbh.de/VfE_html5.mp4')
+            # Create file message
+            msg = Message('file', 'http://www.pdf995.com/samples/pdf.pdf' )
+            
+            # Echo back to user
             msg = Message('text', payload)
+            # Send regular message
```

Comment:
@ben-cunningham they are. In the fact that they arent template messages and are sent exactly the same as a text message would ratherr than contructing buttons and such.